pxr/usdImaging/usdImagingGL/CMakeLists.txt

```diff
@@ -142,3 +152,4411 @@ pxr_build_test(testUsdImagingGLSurfaceShader
+    FAIL 1
+    FAIL_PERCENT 1
+    PERCEPTUAL
+    EXPECTED_RETURN_CODE 0
+    TESTENV testUsdImagingGLUsdSkel
+    ENV
+        USDSKELIMAGING_FORCE_CPU_COMPUTE=0
+)
+
+pxr_register_test(testUsdImagingGLUsdSkelExtCompGPU_arm
+    COMMAND "${CMAKE_INSTALL_PREFIX}/tests/testUsdImagingGLBasicDrawing -offscreen -frameAll -lighting -camlight -shading wireOnSurface -stage arm.usda -times 1 3 7 -write usdSkel_arm_extComp_GPU.png"
+    IMAGE_DIFF_COMPARE
+        usdSkel_arm_extComp_GPU_001.png
+        usdSkel_arm_extComp_GPU_003.png
+        usdSkel_arm_extComp_GPU_007.png
+    FAIL 1
+    FAIL_PERCENT 1
+    PERCEPTUAL
+    EXPECTED_RETURN_CODE 0
+    TESTENV testUsdImagingGLUsdSkel
+    ENV
+        USDSKELIMAGING_FORCE_CPU_COMPUTE=0
+)
+
+pxr_register_test(testUsdImagingGLUsdSkelExtCompGPU_twoBends
+    COMMAND "${CMAKE_INSTALL_PREFIX}/tests/testUsdImagingGLBasicDrawing -offscreen -frameAll -lighting -camlight -shading wireOnSurface -stage twoBends.usda -write usdSkel_twoBends_extComp_GPU.png"
+    IMAGE_DIFF_COMPARE
+        usdSkel_twoBends_extComp_GPU.png
+    FAIL 1
+    FAIL_PERCENT 1
+    PERCEPTUAL
+    EXPECTED_RETURN_CODE 0
+    TESTENV testUsdImagingGLUsdSkel
+    ENV
+        USDSKELIMAGING_FORCE_CPU_COMPUTE=0
+)
+
+pxr_register_test(testUsdImagingGLPopOut
+    COMMAND "${CMAKE_INSTALL_PREFIX}/tests/testUsdImagingGLPopOut --output test test.usda"
+    IMAGE_DIFF_COMPARE
+        test_0.png
+        test_1.png
+        test_2.png
+        test_3.png
+        test_4.png
+        test_5.png
+        test_6.png
+        test_7.png
+        test_8.png
+    FAIL 1
+    FAIL_PERCENT 1
+    PERCEPTUAL
+    EXPECTED_RETURN_CODE 0
+    TESTENV testUsdImagingGLPopOut
+)
+
+pxr_register_test(testUsdImagingGLPopOut_instance
+    COMMAND "${CMAKE_INSTALL_PREFIX}/tests/testUsdImagingGLPopOut --output test_instance test_instance.usda"
+    IMAGE_DIFF_COMPARE
+        test_instance_0.png
+        test_instance_1.png
+        test_instance_2.png
+        test_instance_3.png
+        test_instance_4.png
+        test_instance_5.png
+        test_instance_6.png
+        test_instance_7.png
+        test_instance_8.png
+    FAIL 1
+    FAIL_PERCENT 1
+    PERCEPTUAL
+    EXPECTED_RETURN_CODE 0
+    TESTENV testUsdImagingGLPopOut
+)
+
+pxr_register_test(testUsdImagingGLPopOut_instance_empty
+    COMMAND "${CMAKE_INSTALL_PREFIX}/tests/testUsdImagingGLPopOut --output test_instance_empty test_instance_empty.usda"
+    EXPECTED_RETURN_CODE 0
+    TESTENV testUsdImagingGLPopOut
+)
+
+pxr_register_test(testUsdImagingGLMaterialTag
+    COMMAND "${CMAKE_INSTALL_PREFIX}/tests/testUsdImagingGLBasicDrawing -lighting -cullStyle back -offscreen -frameAll -stage materialTag.usda -write testUsdImagingGLMaterialTag.png"
+    IMAGE_DIFF_COMPARE
+        testUsdImagingGLMaterialTag.png
+    FAIL 1
+    FAIL_PERCENT 1
+    PERCEPTUAL
+    EXPECTED_RETURN_CODE 0
+    TESTENV testUsdImagingGLMaterialTag
+)
+
+pxr_register_test(testUsdImagingGLMaterialX_mxSimple
```

Comment:
We'll need to disable all of the MaterialX tests unless PXR_ENABLE_MATERIALX_SUPPORT has been set.